CHANGELOG.md

```diff
@@ -52,6 +52,130 @@ pay attention to. Medium impact changes are also worth looking at.
 
 <!--releases-->
 
+## 2025-01-01: 2024q4 release
+
+This release covers everything from 2024-10-01 and has been tested with avr-gcc
+v13.2.0 from Upstream and arm-none-eabi-gcc v13.2.1 from xpack.
+
+Breaking changes:
+
+**Protothreads and Resumables are deprecated! Please use Fibers instead!**
+Note that you can disable the deprecation by resetting the option
+`modm:processing:protothread:use_fiber` to `false`, however, you will be limited
+to using GCC12 as GCC13 and newer does not compile our implementation anymore.
+
+A shim is provided to allow the use of Protothreads and Resumables with Fibers
+without any significant modification of the existing code. However you must
+replace the main loop calling all protothreads with the fiber scheduler:
+
+```cpp
+int main()
+{
+    /*
+    while(true)
+    {
+        protothread1.update();
+        protothread2.update();
+    }
+    */
+    modm::fiber::Scheduler::run();
+    return 0;
+}
+```
+
+The `modm:processing:protothread` and `modm:processing:resumable` modules will
+be removed in 2026. Please port your code to use fibers instead. Consult the
```

Comment:
Another >1 year with protothreads? 😱 
```suggestion
be removed by the end of 2025. Please port your code to use fibers instead. Consult the
```

Reply:
No! I meant that 2025q4 is technically in 2026, but this is clearer.

Reply:
I already assumed that and like it, but I think the wording with 2026 might seem unclear to other people.